anagramGroups.js

```diff
@@ -0,0 +1,61 @@
+let strs = ['eat', 'tea', 'tan', 'ate', 'nat', 'bat'];
+
+let group = new Array();
+
+function compareObjects(item1, item2) {
```

Comment:
`compareObjects` are incorrect
![image](https://user-images.githubusercontent.com/8700844/100794790-be3cf680-3426-11eb-9b61-12364b21b22d.png)

Reply:
This function is over complicated. I cannot get an idea behind it. Could you please explain?

Reply:
Yes, I found mistakes and fixed. Thanks.